kong/plugins/prometheus/prometheus.lua

```diff
@@ -200,12 +203,12 @@ local function full_metric_name(name, label_names, label_values)
         label_value = string.sub(label_value, 1, pos - 1)
       end
 
-      if string.find(label_value, "\\", 1, true) then
-        label_value = ngx_re_gsub(label_value, "\\", "\\\\", "jo")
+      if string.find(label_value, slash, 1, true) then
+        label_value = ngx_re_gsub(label_value, reg_slash, double_slash, "jo")
       end
```

Comment:
looks like `reg_slash` is same as `double_slash`, was there no actual replacement?

Reply:
No, `reg_slash ` is a regex, i.e. `\`, but `double_slash` is a plain text `\\`.

Reply:
I get it.